src/WebSite/Bit.Platform.WebSite/Web/Components/Footer.razor

```diff
@@ -1,3 +1,35 @@
-﻿<footer class="footer">
-    <div>© 2022 Bit team. All rights reserved.</div>
+﻿<footer class="footer-container">
+    <section class="footer-section divider">
+        <div class="bit-column">
+            <div class="bit-icon" />
+            <div class="line">Whoooo we are and what we do in bit or why we want to resolve others issus</div>
+        </div>
+        <div class="products-column">
+            <div class="header">Products</div>
+            <a class="line" target="_blank" href="@Urls.TodoTemplate">BitTodoTemplate</a>
+            <a class="line" target="_blank" href="@Urls.Components">BitComponents</a>
+            <a class="line" target="_blank" href="@Urls.Bswup">BitBswup</a>
+        </div>
+        <div class="quick-links-column">
+            <div class="header">Quick links</div>
+            <a class="line" target="_blank" href="@Urls.GithubRepo">Github Repo</a>
+            <a class="line" target="_blank" href="@Urls.GithubOrg">Github Org</a>
+            <a class="line" target="_blank" href="@Urls.GithubIssues">Issues</a>
+        </div>
+        <div class="media-column">
+            <div class="header">Media</div>
+            <a href="@Urls.Website" target="_blank"  class="line">Web site</a>
+            <a href="@Urls.Blog" target="_blank" class="line">Blog</a>
+        </div>
```

Comment:
are these footer items approved to be here?

@ysmoradi & @maryamhdr 
did you check the design?

Reply:
These items are a little different from the design...